### PR TITLE
Remove Kamon instrumentation closes #673

### LINKED ISSUE #673: Spike: Kamon

### PR DESCRIPTION
**Pull request by geoffjentry**

### CODE REVIEW

**Comment by cjllanwarne** [comment]
:+1:

**Comment by geoffjentry** [comment]
@Horneth

File: engine/src/main/scala/cromwell/server/WorkflowManagerSystem.scala [hunk 1]
```diff
@@ -1,20 +1,14 @@
...
-import cromwell.instrumentation.Instrumentation._
 import collection.JavaConversions._
 
 trait WorkflowManagerSystem {
-  Monitor.start()
-
   protected def systemName = "cromwell-system"
-
   protected def newActorSystem(): ActorSystem = ActorSystem(systemName)
-
```
Comment (kshakir) [inline engine/src/main/scala/cromwell/server/WorkflowManagerSystem.scala]: Remove these lines before/after the `def`s all you want. IntelliJ ⌘⌥O is going to put them right back.

Comment (geoffjentry) [inline engine/src/main/scala/cromwell/server/WorkflowManagerSystem.scala]: Intellij is wrong

**Comment by kshakir** [comment]
👍

**Comment by geoffjentry** [comment]
@kshakir just for you

**Pull request merged**